config/vufind/config.ini

```diff
@@ -2246,3 +2246,8 @@ show_solr_options_in_tag_search = false
 description = "The REST API provides access to search functions and records contained in the search index."
 ; URL pointing to a Terms of Service page (optional, default is none):
 ;termsOfServiceUrl = "https://something"
+
+[Sorting]
+; Default sorting in VuFind is case insensitive and does not respect locales. If
```

Comment:
I think you can remove the "is case insensitive and" part.

Reply:
Thanks @demiankatz